lib/Kubernetes/events/kubernetes_event_monitor.rb

```diff
@@ -0,0 +1,35 @@
+class KubernetesEventMonitor
```

Comment:
I think this file should add a `require 'kubeclient'` at the time, so as not to assume it's loaded elsewhere.